src/lib/utils/get-platform-shader-defines.js

```diff
@@ -0,0 +1,41 @@
+import {glGetDebugInfo} from 'luma.gl';
+
+export function checkRendererVendor(debugInfo, vendor) {
+  let result = false;
+  switch (vendor) {
+  case 'nvidia':
+    result = debugInfo.vendor.match(/NVIDIA/i) || debugInfo.renderer.match(/NVIDIA/i);
+    break;
+  case 'intel':
+    result = debugInfo.vendor.match(/INTEL/i) || debugInfo.renderer.match(/INTEL/i);
+    break;
+  case 'amd':
+    result = debugInfo.vendor.match(/AMD/i) ||
+    debugInfo.renderer.match(/AMD/i) ||
+    debugInfo.vendor.match(/ATI/i) ||
+    debugInfo.renderer.match(/ATI/i);
+    break;
+  default:
+    result = false;
+  }
+  return result;
+}
+
+export function getPlatformShaderDefines(gl) {
+  let platformDefines = '';
+  const debugInfo = glGetDebugInfo(gl);
+
+  if (checkRendererVendor(debugInfo, 'nvidia')) {
+    platformDefines += '#define NVIDIA_GPU \n' +
+    '#define NVIDIA_FP64_WA 1\n' +
+    '#define NVIDIA_EQUATION_WA 1';
```

Comment:
- There should be more documentation (in comments) about these flags, either here or at point of use, or both. 
- Also naming is better now, but still confusing (i.e. what is an EQUATION workaround?)
- I think `WA` could be spelled out as `WORKAROUND` - doesn't seem like an obvious abbreviation...